Spanner/src/Connection/Grpc.php

```diff
@@ -1194,6 +1156,40 @@ public function commit(array $args)
         ]);
     }
 
+    /**
+     * @param array $args
+     * @return \Generator
+     */
+    public function batchWrite(array $args)
+    {
+        $databaseName = $this->pluck('database', $args);
+        $mutationGroups = $this->pluck('mutationGroups', $args);
+        $requestOptions = $this->pluck('requestOptions', $args, false) ?: [];
+
+        array_walk(
+            $mutationGroups,
+            fn(&$x) => $x['mutations'] = $this->parseMutations($x['mutations'])
+        );
```

Comment:
This should pass a normal function instead of a shorthand one

```suggestion
        array_walk(
            function ($x, $i) {
                $mutationGroups[$i]['mutations'] = $this->parseMutations($x['mutations']);
            },
            $mutationGroups,
        );
```

OR it should use `array_map`:

```suggestion
        $mutationGroups = array_walk(
            fn($x) => ['mutations' => $this->parseMutations($x['mutations'])] + $x,
            $mutationGroups
        );
```

The way this is using short functions and pass-by-reference with `array_walk` is not the intended use.